notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -2,7 +2,7 @@
   "cells": [
```

Comment:
<p>Change this to "Review results"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>